packages/contracts/contracts/PriceFeed.sol

```diff
@@ -3,566 +3,161 @@
-                    _changeStatus(Status.bothOraclesUntrusted);
-                    return lastGoodPrice;
-                }
-
-                // If Chainlink is broken, remember it and switch to using Tellor
-                _changeStatus(Status.usingTellorChainlinkUntrusted);
-
-                if (_tellorIsFrozen(tellorResponse)) {return lastGoodPrice;}
-
-                // If Tellor is working, return Tellor current price
-                return _storeTellorPrice(tellorResponse);
-            }
-
-            if (_chainlinkIsFrozen(chainlinkResponse)) {
-                // if Chainlink is frozen and Tellor is broken, remember Tellor broke, and return last good price
-                if (_tellorIsBroken(tellorResponse)) {
-                    _changeStatus(Status.usingChainlinkTellorUntrusted);
-                    return lastGoodPrice;
-                }
-
-                // If both are frozen, just use lastGoodPrice
-                if (_tellorIsFrozen(tellorResponse)) {return lastGoodPrice;}
-
-                // if Chainlink is frozen and Tellor is working, keep using Tellor (no status change)
-                return _storeTellorPrice(tellorResponse);
-            }
-
-            // if Chainlink is live and Tellor is broken, remember Tellor broke, and return Chainlink price
-            if (_tellorIsBroken(tellorResponse)) {
-                _changeStatus(Status.usingChainlinkTellorUntrusted);
-                return _storeChainlinkPrice(chainlinkResponse);
-            }
-
-             // If Chainlink is live and Tellor is frozen, just use last good price (no status change) since we have no basis for comparison
-            if (_tellorIsFrozen(tellorResponse)) {return lastGoodPrice;}
-
-            // If Chainlink is live and Tellor is working, compare prices. Switch to Chainlink
-            // if prices are within 5%, and return Chainlink price.
-            if (_bothOraclesSimilarPrice(chainlinkResponse, tellorResponse)) {
-                _changeStatus(Status.chainlinkWorking);
-                return _storeChainlinkPrice(chainlinkResponse);
-            }
-
-            // Otherwise if Chainlink is live but price not within 5% of Tellor, distrust Chainlink, and return Tellor price
-            _changeStatus(Status.usingTellorChainlinkUntrusted);
-            return _storeTellorPrice(tellorResponse);
-        }
-
-        // --- CASE 5: Using Chainlink, Tellor is untrusted ---
-        if (status == Status.usingChainlinkTellorUntrusted) {
-            // If Chainlink breaks, now both oracles are untrusted
-            if (_chainlinkIsBroken(chainlinkResponse, prevChainlinkResponse)) {
-                _changeStatus(Status.bothOraclesUntrusted);
-                return lastGoodPrice;
-            }
-
-            // If Chainlink is frozen, return last good price (no status change)
-            if (_chainlinkIsFrozen(chainlinkResponse)) {
-                return lastGoodPrice;
-            }
+abstract contract LEDLike {
```

Comment:
I've never seen this "Like" syntax before. Is this a replacement for the term "base" for abstract classes? Any reason to not make this an interface instead of an abstract class?